.gitignore

```diff
@@ -1,8 +1,9 @@
 /.installed.cfg
 /.mr.developer.cfg
-/buildout.cfg
+/.vscode
```

Comment:
@reinhardt Let's also remove the `/.vscode` line. That slipped in via my patch even I do not use vscode.